spec.md

```diff
@@ -1722,8 +1845,15 @@ String interpolation
    string % any                 # see String Interpolation
 ```
 
-The operands of the arithmetic operators `+`, `-`, `*`, `//`,`%`, `&`,
-`|` and `^` must both be `int`. The type of the result has type `int`.
+The operands of the arithmetic operators `+`, `-`, `*`, `//`, and `%`,
+must both be numbers (`int` or `float`) but need not have the same type.
```

Comment:
Do we mention somewhere in the spec the difference with Python that booleans are not numbers?

Reply:
Good point. Added:

```
True and False may be converted to the values 0 and 1 using the `int` function,                      
but Booleans are not numbers.                                                                        
```

to Booleans.